.gitlab-ci.yml

```diff
@@ -1,126 +1,21 @@
-#         time podman rmi -f $(podman images -q -f "dangling=true")
-#       else 
-#         echo "LOG INFO No Cleanup needed this time value: $CACHE_USE_PERCENT"
-#       fi
-#   retry: 1
-#   after_script:
-#     - podman logout docker.io
-# 
-# parachains-full-test-set:          &parachains-tests
-#   tags:
-#     - parity-simnetscripts-build
-#   stage:                           test
-#   image:                           quay.io/podman/stable
-#   interruptible:                   true
-#   variables:
-#     # IMAGE_NAME:                docker.io/parity/rococo
-#     IMAGE_NAME:                "docker.io/paritypr/synth-wave"
-#     # IMAGE_TAG:                 rococo-v1
-#     IMAGE_TAG:                 master
-#     COLLATOR_IMAGE_TAG:                 master
-#     # COLLATOR_IMAGE_TAG:        master
-#   before_script:
-#     - echo "'${CI_PIPELINE_SOURCE}' type trigger."
-#     - echo "Image under test '${IMAGE_NAME}:${IMAGE_TAG}'."
-#     - echo "Collator image 'docker.io/paritypr/colander:${COLLATOR_IMAGE_TAG}'."
-#     - test "$DOCKER_SIMNET_USER" -a "$DOCKER_SIMNET_PASS" ||
-#         ( echo "No docker credentials provided"; exit 1 )
-#   script:
-#     - podman pull "docker.io/paritytech/simnetscripts:latest"
-#     - podman pull "${IMAGE_NAME}:${IMAGE_TAG}"
-#     - podman run
-#            --volume "$(pwd)"/testing:/home/nonroot/gurke/testing
-#            --volume /etc/gurke/:/etc/gurke/
-#            paritytech/simnetscripts:latest
-#                   scripts/run-test-environment-manager.sh
-#                    --test-script=../testing/parachains/run_tests.sh
-#                    --image="${IMAGE_NAME}:${IMAGE_TAG}"
-# 
-# push-simnetscripts-image-to-paritytech:
-#   variables:
-#     <<:                            *global-vars
-#     UPSTREAM_BUILD_TRIGGER:        $BUILD_SIMNETSCRIPTS
-#   stage:                           post-merge
-# #  needs:
-# #    - job:                         test-simnetscripts-image
-#   image:                           quay.io/buildah/stable
-#   rules:
-#     - if: '$CI_COMMIT_BRANCH == "main"'
-#     - if: '$UPSTREAM_BUILD_TRIGGER == "true"'
-#   tags:
-#     - parity-simnet-build 
-#   before_script:
-#     - test "$Docker_Hub_User_Parity" -a "$Docker_Hub_Pass_Parity" ||
-#         ( echo "no docker credentials provided"; exit 1 )
-#   script:
-#     - echo "LOG INFO step 7 update paritytech/simnetscripts:latest"
-#     - echo "$Docker_Hub_Pass_Parity" |
-#         buildah login --username "$Docker_Hub_User_Parity" --password-stdin docker.io
-#     - buildah info
-#     - 'buildah pull "docker.io/paritypr/simnetscripts:$CI_COMMIT_SHORT_SHA"'
-#     - 'buildah tag "docker.io/paritypr/simnetscripts:$CI_COMMIT_SHORT_SHA" docker.io/paritytech/simnetscripts:latest'
-#     - 'buildah push --format=v2s2 docker.io/paritytech/simnetscripts:latest'
-#   after_script:
-#     - buildah logout docker.io
-# # dummy
+    - parity-simnetscripts
```

Comment:
why do you need a new tag here?

Reply:
if this is just a trigger, and nothing else, why would you need separate runners?

Reply:
In the future I want to add an image for each simnet component ;
As I mentioned I am not in favor of this monolith design we have now.
I want to leave the door open to have this option, once we solve all major points

Reply:
Dunno about an image for an each component, since together they make a viable product and this way makes it easier to communicate between them, I don't see why them all together is somehow a dirty design. Although, putting some certain static parts like test scripts and configs to a static place would be wiser than storing them in the image.

Reply:
> With current design if some developers want to add more test, 

no one will want to add any tests as long as the project will be this complicated